docs/source/reference/series.rst

```diff
@@ -70,3 +70,10 @@ Missing data handling
    Series.isna
    Series.notna
    Series.dropna
+
+Serialization / IO / Conversion
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. autosummary::
+   :toctree: api/
+
+   DataFrame.to_numpy
```

Comment:
Series.to_numpy?

Also Pandas have this in the "Conversion" section. Maybe we should just follow that.

Reply:
Series has [to_numpy()](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.to_numpy.html)
I agree with you, we can follow Pandas documentation!

Reply:
But you are adding DataFrame.to_numpy doc in series.rst?

Reply:
@garawalid I think you meant `Series.to_numpy` here

Reply:
You are right, it should be Series.to_numpy